airrohr-firmware/airrohr-firmware.ino

```diff
@@ -405,6 +407,181 @@ SoftwareSerial serialGPS(GPS_SERIAL_RX, GPS_SERIAL_TX, false, 512);
 #define serialGPS (Serial2)
 #endif
 
+/*****************************************************************
+ * MHZ19 declarations                                           *
+ *****************************************************************/
+class CMHZ19Sensor
```

Comment:
Please move the sensor implementation into a separate pair of cpp/h files to not clutter the main file too much.. it's large enough already